disperser/dataapi/operator_handler.go

```diff
@@ -221,12 +221,8 @@ func checkServiceOnline(ctx context.Context, serviceName string, socket string,
 	return false, fmt.Sprintf("grpc available but %s service not found at %s", serviceName, socket)
 }
 
-func (oh *OperatorHandler) GetOperatorsStake(ctx context.Context, operatorId string) (*OperatorsStakeResponse, error) {
-	currentBlock, err := oh.indexedChainState.GetCurrentBlockNumber()
-	if err != nil {
-		return nil, fmt.Errorf("failed to fetch current block number: %w", err)
-	}
-	state, err := oh.chainState.GetOperatorState(ctx, currentBlock, []core.QuorumID{0, 1, 2})
+func (oh *OperatorHandler) GetOperatorsStakeAtBlock(ctx context.Context, operatorId string, currentBlock uint32) (*OperatorsStakeResponse, error) {
+	state, err := oh.chainState.GetOperatorState(ctx, uint(currentBlock), []core.QuorumID{0, 1, 2})
```

Comment:
should quorum id here be parameter of function?

Reply:
no it's just fetching entire state (one RPC call anyway)